src/components/PlayerList/PlayerList.js

```diff
@@ -0,0 +1,21 @@
+import ListItem from '../ListItem/ListItem.js'
+import styled from 'styled-components/macro'
+
+const PlayerList = ({ data }) => {
+  return (
+    <StyledList>
+      {data.map(player => {
+        return <ListItem key={player.id} name={player.name} />
+      })}
+    </StyledList>
+  )
+}
+
+const StyledList = styled.div`
```

Comment:
Please use semantic HTML! If you are using the `list` as a name, I would _definitely_ think that this component is an `ol` or `ul`.